pact-request-api/Pact/Core/Command/Client.hs

```diff
@@ -0,0 +1,133 @@
+module Pact.Core.Command.Client where
+
+import Control.Monad
+import Control.Monad.Except
+import Data.ByteString (ByteString)
+import Data.Text (Text)
+import qualified Pact.JSON.Encode as J
+
+import Pact.Core.Capabilities
+import Pact.Core.ChainData
+import Pact.Core.Command.RPC
+import Pact.Core.Command.Types
+import Pact.Core.Command.Util
+import Pact.Core.Command.Crypto
+import Pact.Core.Guards
+import qualified Pact.Core.Hash as PactHash
+import Pact.Core.PactValue
+import Pact.Core.Names
+import Pact.Core.Verifiers
+
+-- CREATING AND SIGNING TRANSACTIONS
```

Comment:
the comment feels misplaced.

Reply:
Agreed. I've totally overhauled the haddocks in this file, and removed this comment.